README.md

```diff
@@ -100,6 +100,7 @@ Note: Browser CORS Restrictions apply. Some browsers may not allow local images
 from other folders, and throw a Security Error instead.
 
 ```js
+  var sequencer = ImageSequencer();
```

Comment:
Since this is already covered in Initializing the sequencer, I think that adding the Initilizing section here as well would do it more informatively.
Does that make sense?

Reply:
@Divy123  Do you mean adding another section titled **Initializing Sequencer**  above the quick usage section?